src/Web/couch-potatoes-web-app/src/components/movie-details/MovieDetailsBottomInformationbox.tsx

```diff
@@ -0,0 +1,98 @@
+import Movie from "../../models/movie";
+import React, {FC} from "react";
+import {Box, Button, Card, CardBody, CardHeader, Heading, Link, Stack, StackDivider, Text} from "@chakra-ui/react";
+import MovieCredits from "../../models/movie_credits";
+
+export interface MovieDetailsComponentProps {
+	movie: Movie | null;
+	themeColor : string ;
+}
+
+export interface MovieDetailsCreditsComponentProps {
+	movieCredits: MovieCredits | null;
+	movie: Movie | null;
+	themeColor : string ;
+}
+
+export const MovieDetailsBottomInformationbox: FC<MovieDetailsCreditsComponentProps> = ({movieCredits,movie,themeColor}) => {
+	return (
+		<>
+
+			<Card>
+				<CardHeader>
+					<Heading size='md'>Information</Heading>
+				</CardHeader>
+				<CardBody>
+					<Stack divider={<StackDivider/>} spacing='2'>
+						<Box >
+							<Heading  textTransform='uppercase' size='sm'>Summary</Heading>
+							<Text pt='2' fontSize='sm' marginBottom={5}>
+								{movie?.summary}
+							</Text>
+							<Stack  direction="row"  spacing='8'>
+								<Stack>
+									<Heading textTransform='uppercase' size='sm'>Director(s)</Heading>
+									<Stack  >
+										{movieCredits && movieCredits?.creditsAsCrew.filter(crew => crew.job === "Director")
+											.map(crew => (
+												<Link
+													color={themeColor}
+													_hover={{color: "grey"}}
+													href="https://example.com"
```

Comment:
Do we have any links we can redirect to now or will it be added later?

Reply:
I am currently looking into using React Router